cmd/commands.go

```diff
@@ -156,20 +154,21 @@ var tmplFlagHelpSmall = `{{ SprintfWhite "######################################
 {{- end }}
 `
 
-
 // ******************************************************************************** //
-var gbHelpCmd = &cobra.Command {
-	Use:					"assist",
+var gbHelpCmd = &cobra.Command{
```

Comment:
@kazishafin — Can you take a look at this pull request?  Notice how there is a huge amount of whitespace changes being committed?  When you do this is make it very difficult to review the pull request.

Your commits should have **only** the relevant changes, not _any_ reformatting.  

If we need to reformat or restructure code we would create a separate commit and pull request for that.

Can you please clean up the branch `kazishafin-68-add-comments` so that this PR **only** contains your additions/changes to comments, and none of the white space?